labels.mss

```diff
@@ -1,11 +1,12 @@
 #country[prio<=3][zoom=3],
 #country[prio<=4][zoom=4],
 #country[zoom>=5][zoom<=10] {
-  text-name: '[name]';
+  text-transform: uppercase;
```

Comment:
Watch out that uppercase does not exist in Arabic alphabet. So if you create visual rules based on that, they will not work with the Arabic style :(

Reply:
I know but for arabic it's not so important I think. 

I could rewrite all the file, but was lazy. Do you think it would be better?

Reply:
> I could rewrite all the file, but was lazy. Do you think it would be better?

You mean rewrite all the file to upper case where needed? If so, I'd say definitively no :). `ŧext-transform` is made for that clearly.

My point was more about using the case as a design semantical parameter, because this will be lost when seeing the Arabic rendering, and thus we may end (little by little) with a French/English style that works well, but is quite ugly and messy in Arabic.
So instead we should use parameters that works in all alphabet covered by our work (like font size, color, font weight…). This point is a really hard one imho, and this is where your designers should help before anywhere else :)